core/src/main/java/com/scalar/db/storage/cassandra/CassandraFactory.java

```diff
@@ -0,0 +1,23 @@
+package com.scalar.db.storage.cassandra;
+
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.api.DistributedStorageFactory;
+import com.scalar.db.config.DatabaseConfig;
+
+public class CassandraFactory implements DistributedStorageFactory {
+  @Override
+  public String getName() {
+    return "cassandra";
```

Comment:
Not a big deal, but is it slightly better to make it constant so that we don't need to hard-code it in other places such as DatabaseConfig and the tests?

Reply:
As discussed, I will leave this as it is, and we can make it constant in the future if needed. Thank you!